frontend/src/scenes/session-recordings/player/inspector/PlayerInspectorControls.tsx

```diff
@@ -242,7 +238,24 @@ function EventsFilterSettingsButton(): JSX.Element {
             icon={<IconUnverifiedEvent />}
             // we disable the filter toggle-all when there are no items
             disabledReason={!hasEventItems ? 'There are no events in this recording' : undefined}
-            hasFilterableItems={hasEventItems}
+            // there is no event upsell currently
+            upsellSideAction={undefined}
+        />
+    )
+}
+
+function CommentsFilterSettingsButton(): JSX.Element {
```

Comment:
![image](https://github.com/user-attachments/assets/d6602d9f-338f-478a-bfa5-a0faa7f9a6a2)
 (always comment on files so we can thread 😅 @veryayskiy )
 
 ideally i'd follow up after this PR
 this is flagged and the various bars already overlap 🙈